pages/sponsor.md

```diff
@@ -0,0 +1,37 @@
+# Sponsorship
+
+## Why Rust Magazine needs sponsorship?
+
+Rust Magazine is a non-profit, community-driven publication that relies on the contributions of volunteers to bring high-quality content to the Rust community. Rust Magazine is seeking sponsorships to:
+
+- Pay our authors for their contributions and inspire more high-quality content.
+- Pay our editors a small salary for their time and effort in maintaining Rust Magazine.
+- Cover the running costs of Rust Magazine, including but not limited to domain fees, email fees, etc.
+
+## How to sponsor Rust Magazine?
+
+You can sponsor Rust Magazine through the [OpenCollective](https://opencollective.com/rustmagazine) platform. Rust Magazine offers several different sponsorship tiers:
+
+- For individuals, you can make a one-time donation with a customized amount, or sponsor Rust Magazine with $1, $5, or $10 per month.
+- For enterprise, you can sponsor us with $25, $50, and $100 per month. Rust Magazine offers different benefits for each tiers.
+- For more detail, please visit <https://opencollective.com/rustmagazine>.
+
+## How does Rust Magazine pay the authors?
+
+```callout, theme:red
+This is a **draft** and is subject to change in the future.
+```
+
+We pay each author according to the following formula: **_min($100, word count / 100 \* $1.25)_**. We pay $1.25 per 100 words, with a maximum payout of $100 per article.
+
+Each author must submit an expense request on the OpenCollective platform. After the admin approves the request, the Fiscal Host will make a payout.
+
+## How Rust Magazine pay the editors?
+
+```callout, theme:red
+This is a **draft** and is subject to change in the future.
+```
+
+Rust Magazine has several editors who take turns being responsible for each Issue. The editor on duty for each Issue will receive a payment of **$50**. If an editor is not on duty for a particular Issue, Rust Magazine will not pay them for that Issue.
```

Comment:
Do we need to make it clear that every issue will have only one editor?

Reply:
Fixed.